Lib/test/_test_multiprocessing.py

```diff
@@ -6801,6 +6801,38 @@ def test_child_sys_path(self):
         self.assertEqual(child_sys_path[1:], sys.path[1:])
         self.assertIsNone(import_error, msg=f"child could not import {self._mod_name}")
 
+    def test_std_streams_flushed_after_preload(self):
+        # gh-135335: Check fork server flushes standard streams after
+        # preloading modules
+        if multiprocessing.get_start_method() != "forkserver":
+            self.skipTest("forkserver specific test")
+
+        # Create a test module in the temporary directory on the child's path
+        # TODO: This can all be simplified once gh-126631 is fixed and we can
+        #       use __main__ instead of a module.
+        dirname = os.path.join(self._temp_dir, 'preloaded_module')
+        init_name = os.path.join(dirname, '__init__.py')
+        os.mkdir(dirname)
+        with open(os.path.join(init_name), "w") as f:
+            cmd = '''if 1:
+                import sys
+                print('stdout', file=sys.stdout)
+                print('stderr', file=sys.stderr)\n'''
+            f.write(cmd)
+
+        name = os.path.join(os.path.dirname(__file__), 'mp_preload_flush.py')
+        env = {'PYTHONPATH': self._temp_dir}
+        rc, out, err = test.support.script_helper.assert_python_ok(name, **env)
+        if rc:
+            support.print_warning("preload flush test failed with stderr:")
+            support.print_warning(err.decode())
+        self.assertEqual(rc, 0)
+
+        # We want to see all the output if it isn't as expected
+        self.maxDiff = None
+        self.assertEqual(err.decode().rstrip(), 'stderr')
+        self.assertEqual(out.decode().rstrip(), 'stdout')
```

Comment:
```suggestion
        self.assertEqual(out.decode().rstrip(), 'stdout')
        self.assertEqual(err.decode().rstrip(), 'stderr')
```

I'd prefer to preserve the same order for reducing cognitive load.

Reply:
I'd normally agree and I had it that way originally, but if the test fails it is probably more useful to see `stderr` than `stdout`, so doing it this way is more likely to show useful diagnostic information on failure. It is a bit surprising for readers though, perhaps a comment would help?

Reply:
IMO, if order really matters it definitely should be some comment on it.
OTOH, you can change the order into `cmd` code, if that's appropriate.
I personally don't like different orders in two places, which one will be changed doesn't matter.